Gemfile

```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'ffaker'
   gem 'pry-byebug'
   gem 'rubocop'
+  gem 'timecop'
```

Comment:
Added for seeding purposes, so that log entries could date back to a month or so ago.